src/main/scala/fm/http/client/Response.scala

```diff
@@ -92,10 +92,13 @@ final class FullResponse(response: HttpResponse, val body: Array[Byte]) extends
 }
 
 object AsyncResponse {
+  // https://stackoverflow.com/questions/15991173/is-the-content-length-header-required-for-a-http-1-0-response/15995101
```

Comment:
This references HTTP 1.0.  PSA is HTTP 1.1.

This should link the the relevant RFC for HTTP 1.1 which, as far as I can tell, would be RFC 7230 Section 3.3.3 #7: https://tools.ietf.org/html/rfc7230#section-3.3.3

```
   7.  Otherwise, this is a response message without a declared message
       body length, so the message body length is determined by the
       number of octets received prior to the server closing the
       connection.
```

Is this only applicable if "Connection: close" is sent since HTTP 1.1 defaults to keep alive?  If so then there would need to be logic for that as well as handling HTTP 1.0 which basically defaults to "Connection: close".